PROJECT

```diff
@@ -5,6 +5,7 @@
 domain: githedgehog.com
 layout:
 - go.kubebuilder.io/v4
+multigroup: true
```

Comment:
Out of curiosity, what does multigroup kubebuilder do?

Reply:
it changes file structure to allow multiple API groups in a single repo, for example it moves api/v1alpha1 to api/gateway/v1alpha1